libs/langchain/langchain/llms/databricks.py

```diff
@@ -43,6 +51,24 @@ class _DatabricksServingEndpointClient(_DatabricksClientBase):
     host: str
     endpoint_name: str
 
+    def __init__(self, **data: Any):
+        super().__init__(**data)
+        self.no_wrap = self._is_external_or_foundation_model(self.endpoint_name)
+
+    def _is_external_or_foundation_model(self) -> bool:
+        """
+        Does this endpoint serve an external or foundation model?
+        """
+        response = self._get(
+            f"https://{self.host}/api/2.0/serving-endpoints/{self.endpoint_name}"
+        )
+        response.raise_for_status()
+        endpoint = response.json()
+        return any(
+            e.get("type", "") in ("EXTERNAL_MODEL", "FOUNDATION_MODEL")
```

Comment:
May want to do `.lower()` and compare to lowercase strings in case this type isn't always capitalized...I worry that it might get uncapitalized at some point

Reply:
updated.